ansible/templates/minigraph_dpg.j2

```diff
@@ -61,6 +61,24 @@
         </a:ManagementIPInterface>
       </ManagementIPInterfaces>
       <ManagementVIPInterfaces xmlns:a="http://schemas.datacontract.org/2004/07/Microsoft.Search.Autopilot.Evolution"/>
+{% if voq_inband_ip is defined %}
+      <VoqInbandInterfaces xmlns:a="http://schemas.datacontract.org/2004/07/Microsoft.Search.Autopilot.Evolution">
+        <a:VoqInbandInterface>
+          <Name>{{ voq_inband_intf }}</Name>
+          <Type>{{ voq_inband_type }}</Type>
+          <a:PrefixStr>{{ voq_inband_ip }}</a:PrefixStr>
+        </a:VoqInbandInterface>
+      </VoqInbandInterfaces>
+{% endif %}
+{% if voq_inband_ipv6 is defined %}
+      <VoqInbandInterfaces xmlns:a="http://schemas.datacontract.org/2004/07/Microsoft.Search.Autopilot.Evolution">
+        <a:VoqInbandInterface>
+          <Name>{{ voq_inband_intf }}</Name>
+          <Type>{{ voq_inband_type }}</Type>
+          <a:PrefixStr>{{ voq_inband_ipv6 }}</a:PrefixStr>
+        </a:VoqInbandInterface>
+      </VoqInbandInterfaces>
+{% endif %}
```

Comment:
Currently, if we have both voq_inband_ipv6 and voq_inband_ip defined, the the generated minigraph.xml would have two VoqInbandInterfaces block. We need to have a single VoqInbandInterfaces block with a VoqInbandInterface block per inband address. So code should be something like:

```
{% if voq_inband_ip is defined or voq_inband_ipv6 is defined %}
      <VoqInbandInterfaces xmlns:a="http://schemas.datacontract.org/2004/07/Microsoft.Search.Autopilot.Evolution">
{% if voq_inband_ip is defined %}
        <a:VoqInbandInterface>
          <Name>{{ voq_inband_intf }}</Name>
          <Type>{{ voq_inband_type }}</Type>
          <a:PrefixStr>{{ voq_inband_ip }}</a:PrefixStr>
        </a:VoqInbandInterface>
{% endif %}
{% if voq_inband_ipv6 is defined %}
        <a:VoqInbandInterface>
          <Name>{{ voq_inband_intf }}</Name>
          <Type>{{ voq_inband_type }}</Type>
          <a:PrefixStr>{{ voq_inband_ipv6 }}</a:PrefixStr>
        </a:VoqInbandInterface>
{% endif %}
      </VoqInbandInterfaces>
{% endif %}
```

Reply:
changed to 

{% if voq_inband_ip is defined or voq_inband_ipv6 is defined %}
      <VoqInbandInterfaces xmlns:a="http://schemas.datacontract.org/2004/07/Microsoft.Search.Autopilot.Evolution">
        <a:VoqInbandInterface>
          <Name>{{ voq_inband_intf }}</Name>
          <Type>{{ voq_inband_type }}</Type>
{% if voq_inband_ip is defined %}
          <a:PrefixStr>{{ voq_inband_ip }}</a:PrefixStr>
{% endif %}
{% if voq_inband_ipv6 is defined %}
          <a:PrefixStr>{{ voq_inband_ipv6 }}</a:PrefixStr>
{% endif %}
        </a:VoqInbandInterface>
      </VoqInbandInterfaces>
{% endif %}